.github/workflows/merge.yml

```diff
@@ -53,3 +55,37 @@ jobs:
         with:
           name: Connector
           path: build/libs/*.jar
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+        if: ${{ github.event_name == 'push' && github.repository == 'odpi/egeria-connector-sas-viya' }}
+      # For releases (ie not main)
+      - name: Build and push (not main merge)
+        if: github.ref != 'refs/heads/main'
+        id: docker_build_release
+        uses: docker/build-push-action@v2 
+        with:
```

Comment:
I wonder if we should add the environment variable here ideally to set the version (though the defaults match for now - it's probably better?) - especially since we already have a version in the script.

See https://github.com/docker/build-push-action/tree/releases/v1#build_args I think
build_args: version=${{ env.RELEASE_VERSION }}

Reply:
This ended up in the with section below as requested.